src/Parser/GDFParser.js

```diff
@@ -0,0 +1,74 @@
+import GeoidGrid from 'Core/Geographic/GeoidGrid';
+import Extent from 'Core/Geographic/Extent';
+
+
+export function getHeaderAttribute(header, attributeName) {
+    const attributeRow = header[header.indexOf(header.find(element => element.includes(attributeName)))].split(' ')
+        .filter(value => value !== '');
+    return attributeRow[attributeRow.length - 1];
+}
+
+
+/**
+ * The `GDFParser` module provides a `[parse]{@link module:GDFParser.parse}` method. This method takes the content of a
+ * GDF file in, and returns a `{@link GeoidGrid}`. the `{@link GeoidGrid}` contains all the necessary attributes and
+ * methods to access the GDF data in iTowns.
+ *
+ * @module GDFParser
+ */
+export default {
+    /**
+     * Parses a GDF file content and returns a corresponding `{@link GeoidGrid}`.
+     *
+     * @param   {string}    gdf                             The content of the GDF file to parse.
+     * @param   {Object}    options                         An object gathering the optional parameters to pass to
+                                                            * the parser.
+     * @param   {Object}    [options.in={}]                 Information on the GDF data.
+     * @param   {string}    [options.in.crs='EPSG:4326']    The Coordinates Reference System (CRS) of the GDF data.
+                                                            * It must be a geographic CRS, and must be given as an EPSG
+                                                            * code.
+     *
+     * @returns {Promise<GeoidGrid>}    A promise resolving with a `{@link GeoidGrid}`, which contains all the necessary
+                                        * attributes and methods to access GDF file data.
+     */
+    parse(gdf, options = { in: {} }) {
+        const rows = gdf.split('\n');
+        const firstMeasureLine = rows.indexOf(rows.find(row => row.includes('end_of_head'))) + 1;
+        const rawHeaderData = rows.slice(0, firstMeasureLine);
+
+        const header = {
+            minX: parseFloat(getHeaderAttribute(rawHeaderData, 'longlimit_west')),
```

Comment:
could you move `parseFloat` in `getHeaderAttribute`?

Reply:
I moved `parseFloat` to `getHeaderAttribute` method.